lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -903,9 +906,10 @@ sub ansible_execute_command {
     my @cmd = ('ansible', $args{host_group},
         "--private-key=$args{sdaf_config_root_dir}/sshkey",
         "--inventory=$args{sap_sid}_hosts.yaml",
+        $args{verbose} ? '-vvv' : '',
```

Comment:
Only a suggestion... no need to change if you don't like it.
There is a function to control verbosity level:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/2f8224c2c80b9133c270e9cf23cc9be02f3835ca/lib/sles4sap/sap_deployment_automation_framework/deployment.pm#L819